src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java

```diff
@@ -888,6 +889,7 @@ private String ignoreLinesDelete(String strSql) {
 
 
     public void clear() {
+        service.getConnection().setProto(new MySQLProtoHandlerImpl());
```

Comment:
should keep this status in new "ProtoHandler" .

Reply:
ok